privaterelay/templates/settings.html

```diff
@@ -0,0 +1,104 @@
+{% extends "base.html" %}
+
+{% load static %}
+{% load ftl %}
+{% ftlconf bundle='privaterelay.ftl_bundles.main' %}
+
+{% block javascript %}
+  <script src="{% static 'js/settings.js' %}"></script>
+{% endblock %}
+
+
+{% block content %}
+
+<main id="settings-main" class=" bg-light dashboard-container " data-api-token="{{ user_profile.api_token }}">
+  <div class="c-meta">
+    <h1 class="mzp-u-title-md c-settings-headline">{% ftlmsg 'settings-headline' %}</h1>
+    <ul class="c-meta--links">
+      <li>
+        <a
+          href="https://support.mozilla.org/"
+          target="_blank"
+          rel="noopener noreferrer"
```

Comment:
Question: Do  we want referral traffic here? (Specifically to another page in our ecosystem)

Reply:
I'm not sure if it's still necessary today, but it's used more as a way to get Firefox to do what it's supposed to do with `noopener`: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
Ahh – Got'cha. We can leave it. 

@groovecoder Is there any sort of telemetry we want to add here?

Reply:
We could add `utm_source=relay.firefox.com` (or `utm_source={{ settings.SITE_ORIGIN }}`) so that SUMO's analytics will show the traffic came from Relay.

Reply:
Let's use https://support.mozilla.org/kb/activate-firefox-relay-hide-your-email-address-alias for now.

Reply:
d2db60d